client/my-sites/earn/memberships/style.scss

```diff
@@ -332,6 +365,19 @@
 	margin-bottom: 1.5em;
 }
 
+.memberships__dialog-sections-price {
+	display: flex;
+	justify-content: flex-start;
```

Comment:
We'll need some mobile styles here.

Reply:
added styles in https://github.com/Automattic/wp-calypso/pull/78959/commits/00e8a7c05f92adfc61e277418bd8f9bd881614b0